### ISSUE TITLE
Q- Does ReShade works with DXVK on?

### ISSUE THREAD

**Issue by ahmed-elsayed2017** [doitsujin/dxvk#784]
I didn't use ReShade before and I want to use it to inject SMAA and try some of the effect that it provides, but I read in their website this:

> The file structure in the game folder (where the game's exe is) is now the following:

    ReShade DLL (can be dxgi.dll, d3d9.dll, opengl32.dll)

So I want make sure if it is going to work with DXVK on.

**Comment by ahmed-elsayed2017** [comment]
I found out that SMAA Injector, FXAA Injector Tool, SweetFX add the same .dll files.

**Comment by doitsujin** [comment]
Never tired it myself, but tbh I doubt this would work on DXVK. Especially when using MinGW builds.

**Comment by ahmed-elsayed2017** [comment]
What about SMAA Injector, FXAA Injection Tool, or SweetFX?

http://www.iryoku.com/smaa/
https://www.guru3d.com/files-details/fxaa-post-process-injection-tool-download.html
http://ceejay.dk/

**Comment by doitsujin** [comment]
These tools all hook into the D3D DLLs, which may or may not work with wine or DXVK. I cannot provide any form of support for those.

**Comment by ahmed-elsayed2017** [comment]
I heard on reddit that ReShade is going to work with DXVK in the next release. Is that true? @doitsujin

**Comment by ahmed-elsayed2017** [comment]
https://www.reddit.com/r/linux_gaming/comments/a2d7xv/reshade_is_now_working_with_latest_dxvk_master/

**Comment by doitsujin** [comment]
I did fix a related bug after someone provided some very useful info about why ReShade in particular was broken before, so it might work now. However, my point stands, I cannot provide support for it.

**Comment by ahmed-elsayed2017** [comment]
It works nicely with DXVK. I think that you will change your mind after using it. Thanks man for your amazing support.

**Comment by fls2018** [comment]
@doitsujin I notice that reshade doesn't work with the proton's 0.94, I'm guessing that particular commit was omitted from it?

**Comment by massatt212** [comment]
I notice proton using a different 0.94 or o could be wrong

**Comment by ahmed-elsayed2017** [comment]
@fls2018 I use DXVK 0.94 and Reshade works just fine.

**Comment by ahmed-elsayed2017** [comment]
@fls2018 The last version of Proton doesn't work with all games. You need to go to Steam Play settings and tick "use this tool instead of game-specific selections from steam" and from the drop list choose the latest Proton version to be able to use this version with all games, and that is not recommended.

**Comment by fls2018** [comment]
Yes I know you can just use the DXVK from here (which still works great) I'm just curious to why it's not in proton.

Just a very minor inconvenience to replace the dll's that's all.
